plugins/periskop/src/components/PeriskopErrorsTable/PeriskopErrorsTable.tsx

```diff
@@ -0,0 +1,190 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import moment from 'moment';
+import { Entity } from '@backstage/catalog-model';
+import { useApi } from '@backstage/core-plugin-api';
+import { useEntity } from '@backstage/plugin-catalog-react';
+import {
+  Content,
+  ContentHeader,
+  Table,
+  TableColumn,
+  Progress,
+  StatusWarning,
+  StatusError,
+  StatusPending,
+  Select,
+  EmptyState,
+  Link,
+} from '@backstage/core-components';
+import Alert from '@material-ui/lab/Alert';
+import useAsync from 'react-use/lib/useAsync';
+import { periskopApiRef } from '../..';
+import { AggregatedError, NotFoundInLocation } from '../../types';
+
+/**
+ * Constant storing Periskop project name.
+ *
+ * @public
+ */
+export const PERISKOP_NAME_ANNOTATION = 'periskop.io/name';
+
+/**
+ * Returns true if Periskop annotation is present in the given entity.
+ *
+ * @public
+ */
+export const isPeriskopAvailable = (entity: Entity): boolean =>
+  Boolean(entity.metadata.annotations?.[PERISKOP_NAME_ANNOTATION]);
+
+const renderKey = (
+  error: AggregatedError,
+  linkTarget: string,
+): React.ReactNode => {
+  return (
+    <Link to={linkTarget} target="_blank" rel="noreferrer">
```

Comment:
you can remove both target and rel, those are injected by our Link (it's part of its benefits that it enforces these things)